README.md

```diff
@@ -16,6 +16,8 @@ Then:
     cd firefox-ui-tests
     python setup.py develop
 
+If you do not have installed virtualenv and virtualenvwrapper you can also use the create_venv.py script to let it automatically create a virtual environmnet for you.
+
```

Comment:
If you already have a virtual environment this script will not be helpful for you, unless you want to replace it. This is really for the CI system and basic starters who do not know how to handle virtual environments. Most of us will even work with virtualenvwrapper, and in that case the location can even not be easily determined by the script. So I don't think it makes sense so cover all those cases.

I will tweak the docs here and add an example.